src/clock-weather-card.ts

```diff
@@ -231,6 +234,7 @@ export class ClockWeatherCard extends LitElement {
             ${this.config.hide_clock ? weatherString : localizedTemp ? `${weatherString}, ${localizedTemp}` : weatherString}
             ${this.config.show_humidity && localizedHumidity ? html`<br>${localizedHumidity}` : ''}
             ${this.config.apparent_sensor && apparentTemp ? html`<br>${apparentString}: ${localizedApparent}` : ''}
+            ${this.config.aqi_sensor && aqi ? html`<br><aqi style="background-color: ${aqiColor}">${aqi} ${aqiString}</aqi>` : ''}
```

Comment:
`this.config.aqi_sensor && aqi !== null` otherwise this will evaluate to false if `aqi` is 0.
or is 0 not a valid aqi value? according to a chart i found, 0 is a valid aqi value